.github/workflows/deploy-nofos.yml

```diff
@@ -39,8 +49,10 @@ jobs:
         uses: ./.github/actions/configure-aws-credentials
         with:
           app_name: nofos
-          environment: ${{ inputs.environment }}
+          environment: ${{ inputs.environment || 'dev' }}
+
+      - name: Publish release
+        run: make release-publish APP_NAME=nofos IMAGE_TAG="$(cat /tmp/commit-hash.txt)"
 
-      - name: Build Docker image
-        working-directory: simpler-grants-pdf-builder
-        run: make build
+      - name: Deploy release
+        run: make release-deploy APP_NAME=nofos ENVIRONMENT=${{ inputs.environment || 'dev' }} IMAGE_TAG="$(cat /tmp/commit-hash.txt)"
```

Comment:
Just to confirm `make release-publish` and  `make release-deploy` are running from your repo, right? We don't need these commands in our Makefile?

Reply:
Confirmed!